internal/driver/tagroot.go

```diff
@@ -0,0 +1,117 @@
+package driver
+
+import (
+	"strings"
+
+	"github.com/google/pprof/internal/measurement"
+	"github.com/google/pprof/profile"
+)
+
+// addLabelNodes adds pseudo-stack-frames "label:value" to each Sample with
```

Comment:
nit: drop dashes in pseudo-stack-frames like elsewhere.

Reply:
Done